### PR TITLE
Task 13.5

### PR DESCRIPTION
**Pull request by AdamPawlinski**
Task 13.5 finished. Please accept merging.

### CODE REVIEW

**Comment by AdamPawlinski** [comment]
Fixed. Please accept final work.

File: index.js [hunk 1]
```diff
@@ -3,26 +3,27 @@ var OSinfo = require('./modules/OSinfo');
...
-        lang = process.env.lang;
-        process.stdout.write(lang + '\n');
-        break;
-      case '/getOSinfo':
-        OSinfo.print();
-        break;
-      default:
-        process.stderr.write('Wrong instruction!\n');
-    }
+  if (input === null) {
```
Comment (matwjablonski) [inline index.js]: use negation ;) `if (!input)`

File: index.js [hunk 2]
```diff
@@ -3,26 +3,27 @@ var OSinfo = require('./modules/OSinfo');
...
+    return
```
Comment (matwjablonski) [inline index.js]: after return add `;`

**Review by matwjablonski — CHANGES_REQUESTED** [review]

**Comment by AdamPawlinski** [comment]
changes made

**Review by matwjablonski — APPROVED** [review]

**Pull request merged**